objects/obj_player/Step_0.gml

```diff
@@ -1,159 +1,159 @@
 #region //Get player input
 
-if (hascontrol)
+if (player_has_control)
 {
-	key_left = keyboard_check(vk_left) || keyboard_check(ord("A"));
-	key_right = keyboard_check(vk_right) || keyboard_check(ord("D"));
-	key_jump = keyboard_check_pressed(vk_space);
+	player_key_left = keyboard_check(vk_left) || keyboard_check(ord("A"));
+	player_key_right = keyboard_check(vk_right) || keyboard_check(ord("D"));
+	player_key_jump = keyboard_check_pressed(vk_space);
 }
 else
 {
-	key_right = 0;
-	key_left = 0;
-	key_jump = 0;
+	player_key_right = 0;
+	player_key_left = 0;
+	player_key_jump = 0;
 }
 
 #endregion
 
 #region //Calculate Horizontal Movement
-walljumpdelay = max(walljumpdelay-1,0);
-if (walljumpdelay == 0)
+player_wall_jump_delay = max(player_wall_jump_delay - 1, 0);
+if (player_wall_jump_delay == 0)
 {
-	var _dir = key_right - key_left;
-	hsp += _dir * hsp_acc;
+	var _dir = player_key_right - player_key_left;
+	player_horizontal_speed += _dir * player_horizontal_speed_acc;
 	if (_dir == 0)
 	{
-		var _hsp_fric_final = hsp_fric_ground;
-		if (!onground) _hsp_fric_final = hsp_fric_air;
-		hsp = scr_approach(hsp,0,_hsp_fric_final);
+		var _horizontal_speed_fric_final = player_horizontal_speed_fric_ground;
+		if (!player_on_ground) _horizontal_speed_fric_final = player_horizontal_speed_fric_air;
+		player_horizontal_speed = scr_approach(player_horizontal_speed, 0, _horizontal_speed_fric_final);
 	}
-	hsp = clamp(hsp,-hsp_walk,hsp_walk);
+	player_horizontal_speed = clamp(player_horizontal_speed, -player_horizontal_speed_walk, player_horizontal_speed_walk);
 }
 //Wall Jump
-if (onwall != 0) && (!onground) && (key_jump)
+if (player_on_wall != 0) && (!player_on_ground) && (player_key_jump)
 {
-	walljumpdelay = walljumpdelay_max;
+	player_wall_jump_delay = player_wall_jump_delay_max;
 	
-	hsp = -onwall * hsp_wjump;
-	vsp = vsp_wjump;
+	player_horizontal_speed = -player_on_wall * player_horizontal_speed_wall_jump;
+	player_vertical_speed = player_vertical_speed_wall_jump;
 	
-	hsp_frac = 0;
-	vsp_frac = 0;
+	player_horizontal_speed_frac = 0;
+	player_vertical_speed_frac = 0;
 }
 
 #endregion
 
 #region //Calculate Vertical Movement
-var _grv_final = grv;
-var _vsp_max_final = vsp_max;
-if (onwall != 0) && (vsp > 0)
+var _grv_final = player_gravity;
+var _vertical_speed_max_final = player_vertical_speed_max;
+if ((player_on_wall != 0) && (player_vertical_speed > 0))
 {
-	_grv_final = grv_wall;
-	_vsp_max_final = vsp_max_wall
+	_grv_final = player_gravity_wall;

```

Comment:
Another variable with no leading word

Reply:
Those are local variables and the beginning `_` is intentional and a naming convention